src/hotspot/share/gc/shenandoah/shenandoahGenerationalHeap.cpp

```diff
@@ -295,6 +295,7 @@ oop ShenandoahGenerationalHeap::try_evacuate_object(oop p, Thread* thread, Shena
 
     if (copy == nullptr) {
       // If we failed to allocate in LAB, we'll try a shared allocation.
+#ifdef KELVIN_ORIGINAL
```

Comment:
This looks like debugging code? Should we back this out?

Reply:
Sorry I forgot to clean this up.  Refined the comment and removed the #ifdef controls